CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - actions, isAboveMobile, isAboveTablet, setNotice to index.js so they can be imported directly
-- $$props.class to Fab, TextArea, Select, Checkbox and List so they can accept global classes
+- $$props.class to Fab, TextArea, Select, Checkbox, Datatable, Dialog.Simple, Dialog.Alert and List so they can accept global classes
```

Comment:
Just wondering...is it always clear which element the class should be applied to?

Reply:
It seems the outermost element in most cases. I am thinking the most common use for components will be to add margin.